javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -575,6 +575,46 @@ public static boolean isIntegralPrimitiveOrBoxed(TypeMirror type) {
     return kind != null && TypeKindUtils.isIntegral(kind);
   }
 
+  /**
+   * Returns true iff the argument is an integral primitive type.
+   *
+   * @param type a type
+   * @return whether the argument is an integral primitive type
+   */
+  public static boolean isIntegralNumericPrimitive(TypeMirror type) {
+    return isIntegralNumericPrimitive(type.getKind());
+  }
+
+  /**
+   * Returns true iff the argument is an integral primitive type.
+   *
+   * @param typeKind a type kind
+   * @return whether the argument is an integral primitive type
+   */
+  public static boolean isIntegralNumericPrimitive(TypeKind typeKind) {
```

Comment:
I think this one needs to be removed because it's the same as the one in TypeKindUtils.